src/context/schemas/action.schema.json

```
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/action.schema.json",
+  "type": "object",
+  "title": "Action",
+  "properties": {
+    "type": { "const": "fdc3.action" },
+    "title": {
+      "type": "string"
+    },
+    "intent": {
+      "type": "string",
+      "$comment": "Should reference an intent type name, such as those defined in the FDC3 Standard'"
+    },
+    "context": {
+      "type": "object",
+      "$ref": "context.schema.json#"
+    },
+    "customConfig": {
+      "type": "object"
+    }
+  },
+  "required": [
+    "type,", "title", "context"
```

```suggestion
    "title", "context"
```
Stray comma + `type` is already required by the context schema